src/app/contexts/UserContext/cookies/index.js

```diff
@@ -1,17 +1,35 @@
 import Cookie from 'js-cookie';
+import pathOr from 'ramda/src/pathOr';
 
 export const getCookiePolicy = () => {
   const POLICY_COOKIE = 'ckns_policy';
 
   return Cookie.get(POLICY_COOKIE) || '000';
 };
 
+export const personalisationEnabled = cookiePolicy =>
+  cookiePolicy && cookiePolicy.length === 3 && cookiePolicy[2] === '1';
+
+const variantCookieConfig = {
+  ukchina: 'chinese',
+  zhongwen: 'chinese',
+};
+
+const getVariantCookieName = service =>
+  pathOr(service, [service], variantCookieConfig);
+
 export const getPreferredVariant = service => {
   if (!service) return null;
-  const VARIANT_COOKIE = `ckps_${service}`;
+  const VARIANT_COOKIE = `ckps_${getVariantCookieName(service)}`;
```

Comment:
Instead of interpolating this cookie name, I'd suggest having a field in the Service config for the relevant services, called
`serviceVariantCookieName` and use that, instead of interpolate in this way. It'll make the code easier to read and maintain, too.

Reply:
🤔 This means we would have to pass the `ServiceContext` to the `UserContext` Provider.  
I think it would be better to have a single reusable config file like https://github.com/bbc/simorgh/pull/5676/files/9fa4cf5b1fb55b08d2c2581d2b2c4444a627f733#diff-8ecc71a1d5a5e3497e99b27934079f7dR12 instead.